### PR TITLE
Worley Noise Node

### PR DESCRIPTION
**Pull request by Pryme8**
How would I combine my branches and PRs?

### CODE REVIEW

**Comment by Pryme8** [comment]
I gotta figure out how to do a toggle for the Manhattan input,

any suggestions?

File: src/Materials/Node/Blocks/worley3DBlock.ts [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+ * block used to Generate a Worley Noise 3D Noise Pattern
+ */
+
+//  Source: https://github.com/Erkaman/glsl-worley
+//  Converted to BJS by Pryme8
+//
+//  Worley Noise 3D
+//  Return vec2 value range of -1.0->1.0, F1-F2 respectivly
+
+export class Worley3DBlock extends NodeMaterialBlock {
```
Comment (deltakosh) [inline src/Materials/Node/Blocks/worley3DBlock.ts]: Would add Noise in the name somehow

**Comment by deltakosh** [comment]
For settings you can look at how I configure the PerturbNormal.invertX and Y properties for isntance

File: src/Materials/Node/Blocks/worley3DBlock.ts (hunk 1 above)
Comment (Pryme8) [reply src/Materials/Node/Blocks/worley3DBlock.ts]: Yes sir!

**Comment by Pryme8** [comment]
> For settings you can look at how I configure the PerturbNormal.invertX and Y properties for isntance

I will do that now, thank you!

**Pull request merged**